### ISSUE TITLE
React 16.3 support

### ISSUE THREAD

**Issue by kaitmore** [DevExpress/testcafe-react-selectors#94]
I'm seeing the following error after upgrading to React 16.3. I'm using testcafe 0.19.1 and testcafe-react-selectors v1.1.0. This appears in all tests using React selectors:

```
 1) An error occurred in Selector code:

      TypeError: Cannot read property 'child' of undefined
```

**Comment by kirovboris** [comment]
Hi @kaitmore,

This issue is fixed in https://github.com/DevExpress/testcafe-react-selectors/pull/90. We're going to publish new version soon.

**Comment by kirovboris** [comment]
I've just published `testcafe-react-selectors@2.0.0`.
